README.md

```diff
@@ -1 +1,8 @@
-# packetsquirrel-payloads
\ No newline at end of file
+# Payload Library for the Packet Squirrel by Hak5
+
+![Packet Squirrel](https://www.hak5.org/wp-content/uploads/2017/10/ps2.jpg "Packet Squirrel")
+
+* [Purchase at HakShop.com](https://hakshop.com/products/packet-squirrel "Purchase at HakShop.com")
+* [Documentation and Wiki](https://wiki.packetsquirrel.com/#!index.md "Documentation and Wiki")
+* [Bash Bunny Forums](https://forums.hak5.org/forum/94-packet-squirrel/ "Packet Squirrel Forums")
```

Comment:
This should probably say "Packet Squirrel" not "Bash Bunny" :)

Reply:
Oops thanks